### PR TITLE
add golang1.17 as a runtime

### PR DESCRIPTION
**Pull request by dgrove-oss**

### CODE REVIEW

**Review by rabbah — APPROVED** [review]

File: golang1.17/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Do not fix the patch level for golang:1.17 to automatically get security fixes.
+FROM golang:1.17-buster
```
Comment (dgrove-oss) [inline golang1.17/Dockerfile:19]: Should we base on Golang:1.17-bullseye instead?

Comment (rabbah) [reply golang1.17/Dockerfile:19]: I dont have an opinion on either.

Comment (dgrove-oss) [reply golang1.17/Dockerfile:19]: It looks to me like buster's LTS EOL is far enough away that we'll have stopped supporting go 1.17 first.  So we can defer the bullseye switch to go 1.18 or 1.19.

**Comment by dgrove-oss** [comment]
Travis seems to really not want to run the check for this PR.  The usual close/re-open game isn't working.  Going to close and open a fresh PR to try again.

**Pull request closed without merging**